packages/database/src/testUtilities/upsertDummyRecord.js

```diff
@@ -79,11 +81,17 @@ export const findOrCreateDummyRecord = async (model, findCriteria, data) => {
 };
 
 export const findOrCreateRecords = async (models, recordsByType) => {
+  const data = [];
```

Comment:
```suggestion
  const data = {};
```
Same below - I think!

Reply:
Oops yeah - great catch! 👏